### ISSUE TITLE
clampRange incorrect with negative values

### ISSUE THREAD

**Issue by domoritz** [uwdata/vega-parser#80]
`clampRange([-10, 20], -20, 100)` should produce `[-10, 20]`
 but produces `[-10, 30]`.

Related #10

```json
{
  "$schema": "https://vega.github.io/schema/vega/v4.json",
  "width": 100,
  "height": 100,
  "marks": [{"type": "text", "encode": {"enter": {"text": {"signal": "clampRange([-10, 20], -20, 100)"}}}}]
}
```

**Comment by jheer** [comment]
Thanks, will investigate.

**Comment by jheer** [comment]
Fixed in vega-parser v3.5.0.
